src/components/App/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Grommet } from 'grommet';
+
+import theme from '../../styles/globalTheme';
+import { WELCOME_PATH, FINAL_PATH, SURVEY_PATH } from '../../constatnts/routes';
+import { useFetch } from '../../hooks/useFetch';
+import { SurveyContext } from '../../context/SurveyContext';
+import GlobalStyle from '../GlobalStyle/GlobalStyle';
+import Survey from '../Survey';
+import FinalPage from '../FinalPage';
+import Main from './Main';
+import Welcome from '../Welcome';
+
+const App = () => {
+  const fetchData = useFetch(null);
```

Comment:
Don't pass null or undefined as function parameter. Just leave that argument empty, and provide default parameter in function declaration. I also suggest to use empty object instead of null, if it fits.